ceph.spec.in

```diff
@@ -102,6 +102,18 @@ BuildRequires:	checkpolicy
 BuildRequires:	selinux-policy-devel
 BuildRequires:	/usr/share/selinux/devel/policyhelp
 %endif
+%if 0%{with make_check}
+%if 0%{?fedora} || 0%{?rhel}
+BuildRequires:	python-cherrypy
+BuildRequires:	python-werkzeug
```

Comment:
the dashboard ceph-mgr plugin depends on cherrypy at run-time, and the restful plugin depends on werkzeug.

so we need them unless they are packaged separately.